packages/terra-demographics-banner/src/DemographicsBanner.scss

```diff
@@ -0,0 +1,71 @@
+/* stylelint-disable  */
+
+@import '~terra-mixins';
+@import './variables';
+@import './mixins';
+
+%terra-demographics-banner {
+  background: $demographics-background;
+  color: #fff;
+  display: -ms-flex;
+  display: flex;
+  margin-bottom: 10px;
+  padding: 10px;
+
+  h1 {
+    margin: 0;
+
+    .terra-DemographicsBanner--preferred-first-name {
+      margin: auto 5px;
+
+      &::before,
+      &::after {
+        content: '"';
+      }
+    }
+  }
+}
+
+.terra-DemographicsBanner {
+  @extend %terra-demographics-banner;
+}
+
+.terra-DemographicsBanner--deceased {
+  @extend %terra-demographics-banner;
```

Comment:
Fixed it here. https://github.com/cerner/terra-core/pull/191/commits/ed668501d50053087fff3d92cfc4675fb7f487ee